active/0000-nested-addon-imports.md

```diff
@@ -0,0 +1,63 @@
+- Start Date: (fill me in with today's date, 2016-08-06)
+- RFC PR: 
+- ember-cli Issue: 
+
+# Summary
+
+The new asset import API, available since ember-cli 2.7.0, offers the possibility to 
+import Ember CLI addons from Ember CLI addons (aka, nested addons).
+This RFC intends to find solutions for typical edge cases.
+
+# Motivation
+
+Since ember-cli 2.7.0, addons can be nested if the nesting floows the following
+scenario:
+* The nested addons do not make use of the property `parent.bowerProperty` to define
+  assets paths because it not always defined (it is not defined in EmberAddon instances)
+* The nested addons do not need to execute a default blueprint.
```

Comment:
`ember install` is what executes that blueprint, so we'd need to abstract that into something that can be called on `npm install` of the overall project.